code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -59,6 +59,17 @@
 
 	switch(M.a_intent)
 		if(I_HELP)
+			if(can_operate(src))
+				if(health >= config.health_threshold_crit)
+					if(src.surgeries.len)
+						for(var/datum/surgery/S in src.surgeries)
+							if(istype(S.get_surgery_step(), /datum/surgery_step/cavity/place_item) || istype(S.get_surgery_step(),/datum/surgery_step/remove_object))

```

Comment:
next step actually EXECUTES the next step, this checks the step you are on. I needed to do this cuase these two surgerys in particlar are open handed. I have to overide the help_shake_act.